cudax/test/utility/basic_any.cu

```diff
@@ -0,0 +1,553 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of CUDA Experimental in CUDA C++ Core Libraries,
+// under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES.
+//
+//===----------------------------------------------------------------------===//
+
+#include <cuda/std/__type_traits/is_standard_layout.h>
+
+#include <cuda/experimental/__utility/basic_any.cuh>
```

Comment:
We conventionally use the public headers for testing
```suggestion
#include <cuda/std/type_traits>

#include <cuda/experimental/utility.cuh>
```

Reply:
there is no `<cuda/experimental/utility.cuh>`. the `__utility/` directory is just a dumping place for implementation details that are for reuse.